doc/developer-notes.md

```diff
@@ -48,8 +48,9 @@ class Class
   The fully-automatic way:
   ========================
```

Comment:
I know this is something that has not been introduced with this PR but I wonder if we could use another kind of mark-up style for this and the next section. Currently the style of the title is the same of the main one. See:

![title](https://user-images.githubusercontent.com/1469203/52413175-20559a80-2ae1-11e9-99af-aecbfce63fd4.png)

What about something like this:

```diff
-  The fully-automatic way:
-  ========================
+  ### The fully-automatic way:
```

Reply:
Will take a look at this also.